index.html

```diff
@@ -14,6 +15,88 @@
 </head>
 
 <body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+  <div class="container">
+
+
+  <nav class="nav">
+
+    <li><a href="services.html">Services</a></li>
+    <li>Product</li>
+    <li>Vision</li>
+    <li>Features</li>
+    <li>About</li>
+    <li>Contact</li>
+<img class="logo-img" src="img/logo.png" alt="Great Idea! Company logo.">
```

Comment:
I would recommend putting this `<img>` element at the top - for logical organization. This may be the reason for your nav being a little off because from a computers point of view, the way you have it written, the image will naturally display at the bottom. Now when you adjust it, it will shift the `<li>`elements as well.